### PR TITLE
add moons and IoT blog post

### PR DESCRIPTION
**Pull request by drbeefsupreme**
This is the first draft of a post about moons and IoT. I wrote it in one sitting
and have not yet edited it so it is not ready for publication. I'd
appreciate feedback.

### CODE REVIEW

**Comment by drbeefsupreme** [comment]
Oh yeah this needs an image still too, if anyone has suggestions.

File: content/blog/iot.md [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+### Future lunar cosmotechnics
+
+We now polish off the old crystal ball and imagine how the usage of Martian
+lunar technology may develop over time once it becomes widely exposed to the
+evolution-inducing forces of industry and culture. This section is pure
+speculation, and I only aim to paint a small glimpse of a potential future
+rather than a prescriptive and exhaustive one. This picture will necessarily be
+biased by my personal proclivities and expertise, or lack thereof.
+
+Any discuss on the future of moons is incomplete without mentioning the
```
Comment (philipcmonk) [inline content/blog/iot.md]:
```suggestion
Any discussion on the future of moons is incomplete without mentioning the
```

File: content/blog/iot.md [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+of this task, which is sometimes more generally known as [data
+fusion](https://en.wikipedia.org/wiki/Data_fusion).
+
+Cryptographic techniques such as [(composable) zero-knowledge
+proofs](https://en.wikipedia.org/wiki/Zero-knowledge_proof), [secure multi-party
+computation](https://en.wikipedia.org/wiki/Secure_multi-party_computation),
+[ring signatures](https://en.wikipedia.org/wiki/Ring_signature), and even
+[homomorphic encryption](https://en.wikipedia.org/wiki/Homomorphic_encryption)
+offer a variety of ways in which data may be shared in an IoT context with
+mathematical guarantees on how much privacy is maintained. This are perhaps less
```
Comment (philipcmonk) [inline content/blog/iot.md]:
```suggestion
mathematical guarantees on how much privacy is maintained. This is perhaps less
```

**Review by philipcmonk — COMMENTED** [review]
Great post!

**Comment by drbeefsupreme** [comment]
I am happy with where its at now. Edits are mostly just grammar and clarifications, and a few minor additions.

**Comment by jalehman** [comment]
Looks like the build needs to be fixed:
Screen Shot 2021-04-23 at 2 19 35 PM

**Comment by drbeefsupreme** [comment]
> Looks like the build needs to be fixed:
> Screen Shot 2021-04-23 at 2 19 35 PM

This commit fixed this https://github.com/urbit/urbit.org/pull/600/commits/e7636ba628c1245da15284919e54fa5c70a22ce5

**Comment by drbeefsupreme** [comment]
This is waiting on feedback from a couple more people that aren't on github.

**Comment by joemfb** [comment]
"Vint Cerf" and "unwitting" probably don't belong in the same sentence ;)

**Comment by drbeefsupreme** [comment]
Noted! I obviously meant that he didn't mean to say anything about Urbit, but it certainly can come off the wrong way.

**Review by urcades — APPROVED** [review]
I support this going out asap

**Comment by drbeefsupreme** [comment]
I'm planning on another round of edits tomorrow.

**Comment by drbeefsupreme** [comment]
A lot of edits thanks to Matt Southley (not on github), mostly just additional clarifications and grammar improvements. I also made it much more clear what what meant by Earth/Mars (and replaced most references to "Mars" with "Urbit"), and included some assumptions on what kind of hardware I'm imagining for the future.

I think at this point I've gotten feedback from enough folks that its good to go.

**Pull request merged**